Allow 5/6/7/M extras to be scrambled on an unsolved puzzle (#14)
thewca/wca-regulations-january-2026: wca-regulations.md

## <article-4><scrambling><scrambling> Article 4: Scrambling

- 4a) A scrambler applies scramble sequences to the solved puzzles.

>     - 4a1) Exception: for extra attempts for 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube and Megaminx, the scramble sequence may be applied to an unsolved puzzle, at the discretion of the WCA Delegate. This exception only applies to scramble sequences for extra attempts.

Author: ```suggestion
    - 4a1) Exception: for extra attempts for 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube and Megaminx, the scramble sequence may be applied to an unsolved puzzle, at the discretion of the WCA Delegate. This exception only applies to scramble sequences for extra attempts. When applying a scramble sequence to an unsolved puzzle, the scramble may begin in any orientation, the usual requirements in [Regulation 4d](regulations:regulation:4d) do not apply.
```

>     - 4a1) Exception: for extra attempts for 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube and Megaminx, the scramble sequence may be applied to an unsolved puzzle, at the discretion of the WCA Delegate. This exception does not apply to scramble sequences which are not used for extra attempts.

Reviewer: The part _This exception does not apply to scramble sequences which are not used for extra attempts._ is redundant since the first sentence already says _for extra attempts_
Additionally, the **double negative** (or triple, since it is an exception?) is super confusing. If it needs to be kept, simplify it to: _This exception only applies to scramble sequences for extra attempts_ (or similar)

>     - 4a1) Exception: for extra attempts for 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube and Megaminx, the scramble sequence may be applied to an unsolved puzzle, at the discretion of the WCA Delegate. This exception does not apply to scramble sequences which are not used for extra attempts.

Author: ```suggestion
    - 4a1) Exception: for extra attempts for 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube and Megaminx, the scramble sequence may be applied to an unsolved puzzle, at the discretion of the WCA Delegate. This exception only applies to scramble sequences for extra attempts.
```
